Python/ceval.c

```diff
@@ -1733,6 +1733,14 @@ _PyEval_EvalFrameDefault(PyFrameObject *f, int throwflag)
                 exc = POP(); /* exc */
                 /* fall through */
             case 0:
+#if defined(_M_ARM) && !defined(DEBUG)
```

Comment:
I think this link says this is fixed in VS 15.8.  Does that seem right?
https://developercommunity.visualstudio.com/content/problem/345767/arm-optimization-change-function-parameter-data-al.html

Reply:
Either fixed in 15.9 or 16.0 - "Fixed pending release" means in a release that hasn't gone out yet, but it was set a while ago so maybe it just wasn't updated properly (they're mostly updated by an automatic process using internal bugs, so can be missed if the links aren't right)

Reply:
Removed optimization workaround